contrib/acpid/files/acpid

```diff
@@ -0,0 +1,6 @@
+type            = process
+command         = /usr/bin/acpid -f
+logfile         = /var/log/acpid.log
```

Comment:
the acpid manpage says it logs everything to syslog, is this needed?

Reply:
with -f it logs to stderr

```c
	/* open the log */
	if (foreground)
		log_to_stderr = 1;
	else
		open_log();
```

generally speaking though, it's nicer to not put it in syslog even if it wasn't stderr